doc/ansible/dev/dependent_watches.md

```diff
@@ -30,4 +30,117 @@ This is an example of a watches file with the `watchDependentResources` field se
   manageStatus: False
   watchDependentResources: True
 
-```
\ No newline at end of file
+```
+
+### Make existing resources dependent
+
+It is strongly recommended to use dependent watches, resource dependency
+should not be an issue for normal operations. However, if your operator
+deployed resources while [dependent watches were
```

Comment:
Is it while dependent watches were disabled, or when the ownerReference injection is turned off? My impression was that if dependentWatches is disabled, we still should be injecting ownerReferences, we just won't be watching those owned resources.